### PR TITLE
po: Use Weblate instead of Zanata

### LINKED ISSUE #856: please migrate to the new Fedora translation platform
Hello, the Fedora project migrates its translation platform to Weblate [1].

This tool directly interact with your git repository, and requires us to know:
* [mandatory] which branch is your development branch?
* [mandatory] have you merged latest translation from Zanata and locked the project?
* [info] Weblate will handle updates when pot file changes, don't edit po files for this [2]
* [optional] what is the license of translation? (please use a code from https://spdx.org/licenses/)
* [optional] do you have any announcement/warning you would like to display to the translators? (it will be displayed in Weblate)
* [optional] do you need us to activate any specific checks? (this is a setting per component [3])
* [optional] do you need us to automatically detect new translation files? (typical usecase: website translation with one translation file per page)

Please note:
* For github hosted projects, Weblate open pull request. For other project, you'll have to add a key to allow commits.
* In Weblate's vocable, one project is a group of component. Each component is a translation file. You can have many projects or many components or both.
* You can change your mind over time, just reach trans@lists.fedoraproject.org

[1] https://communityblog.fedoraproject.org/fedora-localization-platform-migrates-to-weblate/ 
[2] https://docs.weblate.org/en/latest/admin/continuous.html#avoiding-merge-conflicts
[3] https://docs.weblate.org/en/latest/user/checks.html#translation-checks

### PR DESCRIPTION
**Pull request by marusak**
Fixes #856

I tried it on manual refresh:
- Updated `.pot` file - https://github.com/marusak/cockpit-composer-weblate/commit/188048db877bfba329359a7144ed9526e5c8529a
- Created PR - https://github.com/marusak/welder-web/pull/3 (ignore those two more commits. Also it is rather noisy, but thats because this is the first refresh)

 * [x] po-refresh

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,18 +12,28 @@ else
...
+WEBLATE_REPO_BRANCH=master
+
 all: npm-install
 	NODE_ENV=$(NODE_ENV) $(BUILD_RUN)
 
-npm-install-zanata: npm-install
-	npm install --no-save zanata-js
+$(WEBLATE_REPO):
+	git clone --depth=1 $(WEBLATE_REPO_URL) $(WEBLATE_REPO)
+	git -C $(WEBLATE_REPO) checkout $(WEBLATE_BRANCH)
```
Comment (henrywang) [inline Makefile]: Nice! I love this method. Thanks @marusak! 
One comment here. `$(WEBLATE_BRANCH)` should be `$(WEBLATE_REPO_BRANCH)`

Comment (marusak) [reply Makefile]: Indeed, thanks for spotting! Fixed

**Comment by cockpituous** [comment]
po-refresh in progress on 2-cockpit-7.
Log: https://209.132.184.41:8493/logs/po-refresh-867-20200108-135858/

**Comment by cockpituous** [comment]
Command '['make', 'upload-pot', 'clean-po', 'download-po']' returned non-zero exit status 2.: https://209.132.184.41:8493/logs/po-refresh-867-20200108-135858/

**Comment by cockpituous** [comment]
po-refresh in progress on 4-cockpit-9.
Log: https://209.132.184.41:8493/logs/po-refresh-867-20200108-143232/

**Comment by cockpituous** [comment]
po-refresh  done: https://github.com/cockpituous/welder-web/commits/po-refresh-20200108-143333

**Comment by marusak** [comment]
Refresh was successfull, but the `.pot` file is incorrect! https://github.com/weldr/cockpit-composer-weblate/commit/f1a09220fbcbe7c8629ffc596b429d4e5c951012

**Comment by marusak** [comment]
I reverted it in the weblate repo, and now will look why is this happening.

**Comment by cockpituous** [comment]
po-refresh in progress on 1-cockpit-9.
Log: https://209.132.184.41:8493/logs/po-refresh-867-20200108-144545/

**Comment by marusak** [comment]
Ok, so the second refresh went correct (I had bug in the commands).
https://github.com/weldr/cockpit-composer-weblate/commit/658d241063aae6506e8e18757ee696a71d41161c

Again, it correctly picked Turkish as a new language.

File: Makefile [hunk 2]
```diff
@@ -32,6 +32,7 @@ po-push: po/cockpit-composer.pot $(WEBLATE_REPO)
 	git -C $(WEBLATE_REPO) push
 
 po/cockpit-composer.pot: npm-install
+	NODE_ENV=$(NODE_ENV) npm run translations:extract
```
Comment (martinpitt) [inline Makefile]: This looks like it should be in the first commit, but not a biggie.

**Pull request merged**